can/util.py

```diff
@@ -182,9 +182,9 @@ def load_config(path=None, config=None):
         if key not in config:
             config[key] = None
 
-    # this is done later too but better safe than sorry
-    if config['interface'] == 'socketcan':
-        config['interface'] = choose_socketcan_implementation()
+    # deprecated socketcan types
```

Comment:
Maybe print a warning to make users switch to simply `socketcan´.

Reply:
Good idea. Yes we should eventually remove this but first a logged warning is good, then maybe a [`DeprecationWarning`](https://docs.python.org/3/library/exceptions.html#DeprecationWarning).

Reply:
Could you please add a comment like this one https://github.com/hardbyte/python-can/commit/07be6b5aa0fd6bcf84e24d37b76c8866120920f8